includes/in-person-payments/templates/html-in-person-payment-receipt.php

```diff
@@ -93,11 +93,24 @@ function format_price_helper( array $product, string $currency ): string {
 			line-height: 2px;
 		}
 
+		.branding-logo {
+			max-width: 250px;
+			margin: 20px auto;
+		}
+
+		#credentials {
+			font-size: 7px;
+			padding-top: 5px;
+		}
+
 	</style>
 </head>
 <body>
 	<div class="receipt">
 		<div class="receipt__header">
+			<?php if ( isset( $branding_logo['file_content'] ) && isset( $branding_logo['content_type'] ) && '' !== $branding_logo['file_content'] && '' !== $branding_logo['content_type'] ) { ?>
```

Comment:
Perhaps [`empty`](https://www.php.net/empty) can be used here to shorten the conditions?